megamek/src/megamek/common/Report.java

```diff
@@ -334,10 +334,16 @@ public void addDesc(Entity entity) {
             if ((indentation <= Report.DEFAULT_INDENTATION) || showImage) {
                 imageCode = "<span id='" + entity.getId() + "'></span>";
             }
-            add("<font color='0xffffff'><a href=\"" + ENTITY_LINK + entity.getId()
-                    + "\">" + entity.getShortName() + "</a></font>", true);
-            add("<B><font color='" + entity.getOwner().getColour().getHexString(0x00F0F0F0) + "'>"
-                    + entity.getOwner().getName() + "</font></B>");
+            String name = "<font color='0xffffff'><a href=\"" + ENTITY_LINK + entity.getId()
+                    + "\">" + entity.getShortName() + "</a></font>";
+            String color = entity.getOwner().getColour().getHexString(0x00F0F0F0);
+
+            if ((entity.getCrew().getSize() >= 1) && !entity.getCrew().getNickname().isBlank()) {
```

Comment:
Can't the first be !isEmpty?

Reply:
Yes. I cribbed it from EntitySprite, I'll change it there too.

Reply:
Oh, maybe not. getCrew returns a Crew, which is not a collection. There is no Crew.isEmpty()